go-controller/pkg/ovn/egressip.go

```diff
@@ -1147,12 +1147,15 @@ func (oc *Controller) deleteEgressIPAssignments(name string, statusesToRemove []
 			return err
 		}
 		for podKey, podStatus := range oc.eIPC.podAssignment {
-			delete(podStatus.egressStatuses, statusToRemove)
-			podNamespace, podName := getPodNamespaceAndNameFromKey(podKey)
-			if err := oc.eIPC.addPerPodGRSNAT(podNamespace, podName, podStatus.podIPs); err != nil {
-				return err
+			if _, ok := podStatus.egressStatuses[statusToRemove]; ok {
```

Comment:
yeah I tried to do this a few months ago but there were concerns that it would generate too much noise in larger setups.  I will keep it at loglevel 2 if someone wants to still tune this in their setups.